cmd/zpool/zpool_main.c

```diff
@@ -2389,6 +2392,9 @@ zpool_do_import(int argc, char **argv)
 				mntopts = optarg;
 			}
 			break;
+		case 'p':
+			flags |= ZFS_IMPORT_SCRUB_PAUSE;
+			break;
```

Comment:
For pool recovery, importing readonly is absolutely the best option to start with.  In fact, if scrubs were automatically paused for readonly pools that would address my use case.

I see your point about this option being somewhat arbitrary.  I'm not opposed to dropping it particularly if we made the suggested readonly tweak.  But perhaps @alek-p  has another use case in mind?